Geometry/VeryForwardGeometryBuilder/plugins/CTPPSGeometryESModule.cc

```diff
@@ -174,12 +174,18 @@ CTPPSGeometryESModule::buildDetGeomDesc( DDFilteredView* fv, DetGeomDesc* gd )
       newGD->setGeographicalID( TotemRPDetId( arm, station, rp, detector ) );
     }
 
-    // strip RPs
-    else if ( name == DDD_TOTEM_RP_RP_NAME ) {
-      const unsigned int decRPId = fv->copyno();
+    // strip and pixels RPs
+    else if ( name == DDD_TOTEM_RP_RP_NAME || name == DDD_CTPPS_PIXELS_RP_NAME) {
+    uint32_t decRPId = fv->copyno();
```

Comment:
DDFilteredView returns int: https://cmssdt.cern.ch/lxr/source/DetectorDescription/Core/interface/DDFilteredView.h#0054

Reply:
I can add a casting at that line.